tests/testthat/test-backend-.R

```diff
@@ -101,11 +101,14 @@ test_that("lead and lag translate n to integers", {
 
 # strings -----------------------------------------------------------------
 
-test_that("can translate case insensitive like", {
+test_that("can only translate case sensitive str_like", {
   local_con(simulate_dbi())
-  test_translate_sql(str_like(x, "abc"))
+  expect_equal(test_translate_sql(str_like(x, "abc", ignore_case = FALSE)),
+               sql("`x` LIKE 'abc'"))
+  expect_equal(test_translate_sql(str_like(x, "ABC", ignore_case = FALSE)),
+               sql("`x` LIKE 'ABC'"))
```

Comment:
Can you please format these like https://style.tidyverse.org/syntax.html#long-lines ?

Reply:
sure, have changed this in the latest commit to follow the guidelines